### PR TITLE
fix: improvement in Use @if and @else to Add Logic To Your Styles module

### LINKED ISSUE #50131: Improvement in Use @if and @else to Add Logic To Your Styles module
### Describe the Issue

- ### In [Use @if and @else to Add Logic To Your Styles](https://www.freecodecamp.org/learn/front-end-development-libraries/sass/use-if-and-else-to-add-logic-to-your-styles) module, currently, the instructions happens to be as follows:

    The `@if` directive in Sass is useful to test for a specific case - it works just like the `if` statement in JavaScript.
    
    ```scss
    @mixin make-bold($bool) {
      @if $bool == true {
        font-weight: bold;
      }
    }
    ```
    
    And just like in JavaScript, `@else if` and `@else` test for more conditions:
    
    ```scss
    @mixin text-effect($val) {
      @if $val == danger {
        color: red;
      }
      @else if $val == alert {
        color: yellow;
      }
      @else if $val == success {
        color: green;
      }
      @else {
        color: black;
      }
    }
    ```
    Create a mixin called `border-stroke` that takes a parameter `$val`. The mixin should check for the following conditions using `@if`, `@else if`, and `@else`:
    
    ```scss
    light - 1px solid black
    medium - 3px solid black
    heavy - 6px solid black
    ```
    
    If `$val` is not `light`, `medium`, or `heavy`, the border should be set to `none`.

- ### Instead of this, the instruction should be as follows:

  The `@if` directive in Sass is useful to test for a specific case - it works just like the `if` statement in JavaScript.
  
  ```scss
  @mixin make-bold($bool) {
    @if $bool == true {
      font-weight: bold;
    }
  }
  ```
  
  And just like in JavaScript, `@else if` and `@else` directives test for more conditions:
  
  ```scss
  @mixin text-effect($val) {
    @if $val == danger {
      color: red;
    }
    @else if $val == alert {
      color: yellow;
    }
    @else if $val == success {
      color: green;
    }
    @else {
      color: black;
    }
  }
  ```
  
  Create a mixin called `border-stroke` that takes a parameter `$val`. The mixin should check for the following conditions using `@if`, `@else if`, and `@else` directives:
  
  ```scss
  light - 1px solid black
  medium - 3px solid black
  heavy - 6px solid black
  ```
  
  If the value of the `$val` parameter is not `light`, `medium`, or `heavy`, then the `border` property should be set to `none`.

### Affected Page

https://github.com/freeCodeCamp/freeCodeCamp/blob/main/curriculum/challenges/english/03-front-end-development-libraries/sass/use-if-and-else-to-add-logic-to-your-styles.md

### Expected behavior

In [Use @if and @else to Add Logic To Your Styles](https://www.freecodecamp.org/learn/front-end-development-libraries/sass/use-if-and-else-to-add-logic-to-your-styles) module, the instruction should be as follows:

  The `@if` directive in Sass is useful to test for a specific case - it works just like the `if` statement in JavaScript.
  
  ```scss
  @mixin make-bold($bool) {
    @if $bool == true {
      font-weight: bold;
    }
  }
  ```
  
  And just like in JavaScript, `@else if` and `@else` directives test for more conditions:
  
  ```scss
  @mixin text-effect($val) {
    @if $val == danger {
      color: red;
    }
    @else if $val == alert {
      color: yellow;
    }
    @else if $val == success {
      color: green;
    }
    @else {
      color: black;
    }
  }
  ```
  
  Create a mixin called `border-stroke` that takes a parameter `$val`. The mixin should check for the following conditions using `@if`, `@else if`, and `@else` directives:
  
  ```scss
  light - 1px solid black
  medium - 3px solid black
  heavy - 6px solid black
  ```
  
  If the value of the `$val` parameter is not `light`, `medium`, or `heavy`, then the `border` property should be set to `none`.

### System

- Device: Laptop
- OS: macOS 13.0.1
- Browser: Google Chrome
- Version: 107.0.5304.110

### PR DESCRIPTION
**Pull request by anmxls**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #50131

### CODE REVIEW

File: curriculum/challenges/english/03-front-end-development-libraries/sass/use-if-and-else-to-add-logic-to-your-styles.md [hunk 1]
```diff
@@ -39,15 +39,15 @@ And just like in JavaScript, `@else if` and `@else` test for more conditions:
...
+Create a mixin called `border-stroke` that takes a parameter `$val`. The mixin should check for the following conditions using `@if`, `@else if`, and `@else` directives:
 
 ```scss
 light - 1px solid black
 medium - 3px solid black
 heavy - 6px solid black
 ```
 
-If `$val` is not `light`, `medium`, or `heavy`, the border should be set to `none`.
+If the value of the `$val` parameter is not `light`, `medium`, or `heavy`, then the `border` property should be set to `none`.
```
Comment (lasjorg) [inline curriculum/challenges/english/03-front-end-development-libraries/sass/use-if-and-else-to-add-logic-to-your-styles.md:50]: "If the value of the `$val` parameter" sounds a bit awkward. Maybe instead?

```suggestion
If the `$val` parameter value is not `light`, `medium`, or `heavy`, then the `border` property should be set to `none`.
```

File: curriculum/challenges/english/03-front-end-development-libraries/sass/use-if-and-else-to-add-logic-to-your-styles.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ The `@if` directive in Sass is useful to test for a specific case - it works jus
 }
 ```
 
-And just like in JavaScript, `@else if` and `@else` test for more conditions:
+And just like in JavaScript, `@else if` and `@else` directives test for more conditions:
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/03-front-end-development-libraries/sass/use-if-and-else-to-add-logic-to-your-styles.md:21]:
```suggestion
And just like in JavaScript, the `@else if` and `@else` directives test for more conditions:
```

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

**Review by lasjorg — APPROVED** [review]
LGTM

**Review by naomi-lgbt — APPROVED** [review]
Works for me!

**Pull request merged**